ZuluContent/Engines/Craft/Core/CraftItem.cs

```diff
@@ -20,12 +19,14 @@ public enum ConsumeType
         None
     }
 
-    public interface ICraftable
+    public interface ICraftable : IEnchanted
```

Comment:
Yeah this is getting a bit crazy, we shouldn't be putting `IEnchanted` on everything, it's really only meant to be used for equippable items. We can make an exception in the single click handler to check for `ICraftable`'s `MarkQuality`.

Just drop the `IEnchanted` inheritance and serialize `MarkQuality` normally in the `Serialize/Deserialize` inside of `ICraftable` which the implementing classes should already be using in their own serialization methods, e.g.

https://github.com/zuluhotelaustralia/zuluhotel/blob/137a3af3cbf674d8d132735d93c9fa2941aaea1e/ZuluContent/Items/Containers/Bridle.cs#L152-L163

Reply:
Alright yeah fair point!